vtex/toolbelt
# Add tester-hub retries and ignore cypress generated assets

#### What is the purpose of this pull request?
Improve `vtex test e2e` experience.

#### What problem is this solving?
- when `tester-hub` isn't up yet, the `vtex test e2e` command times out after 15 seconds and throws an error to the user. This PR adds a retry count, giving `tester-hub` up to a minute to start
- - It might still time out due to `cypress-e2e-worker` not being up: https://github.com/vtex/tester-e2e-cypress/pull/8
- should someone run Cypress locally, it might generate videos/screenshots which we shouldn't upload on `link`/`publish`

#### How should this be manually tested?
- Link `tester-hub`, wait 10 minutes for it to be brought down and run `vtex test e2e`, it should not time out.
- Add any files to `cypress/(videos|screenshots)` and make sure they are not listed upon `vtex link --verbose` 

#### Types of changes

#### Chores checklist
